docs/authoring.md

```diff
@@ -0,0 +1,108 @@
+The following item group demonstrates the needed modifications to the project file (1. above):
+
+```
+<ItemGroup>
+    <!-- the winmd -->
+    <Reference Include="PosnLibrary">
+      <HintPath>..\PosnLibrary\Generated Files\PosnLibrary.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+    </Reference>
+    
+    <!-- the runtimeconfig.json -->
+    <None Include="WinRT.Host.runtimeconfig.json">
+      <DeploymentContent Condition="'$(Configuration)|$(Platform)'=='Release|x64'">true</DeploymentContent>
+      <DeploymentContent Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">true</DeploymentContent>
+    </None>
+    
+    <!-- the manifest -->
+    <Manifest Include="PosnConsole.exe.manifest">
+      <DeploymentContent>true</DeploymentContent>
+    </Manifest>
+</ItemGroup> 
+```
+
+### Manifest and RuntimeConfig
+You'll need to author some files to assist the hosting process by the native app: `YourNativeApp.exe.manifest` and `WinRT.Host.runtimeconfig.json`. 
+For information on writing these, see the [hosting docs](https://github.com/microsoft/CsWinRT/blob/master/docs/hosting.md).
+
+### Copying DLLs Target
+
+In your C++ app, add a `Directory.Build.targets` file that copies over the necessary DLLs: 
+```
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <PrepareForRunDependsOn>CopyHostAssets;$(PrepareForRunDependsOn)</PrepareForRunDependsOn>
+  </PropertyGroup>
+  
+  <Target Name="CopyHostAssets">
+    <Copy SourceFiles="Path\To\Nuget\Packages\microsoft.windows.cswinrt\DownloadedVersion\native\$(Platform)\WinRT.Host.dll"
+          DestinationFolder="$(OutDir)" 
+          UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
+    
+    <Copy SourceFiles="Path\To\Nuget\Packages\microsoft.windows.cswinrt\DownloadedVersion\lib\net5.0\WinRT.Host.Shim.dll"
+          DestinationFolder="$(OutDir)" 
+          UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
+
+    <Copy SourceFiles="..\PosnLibrary\bin\$(Platform)\$(Configuration)\net5.0-windows10.0.19041.0\WinRT.Runtime.dll" 
+          DestinationFolder="$(OutDir)" 
+          UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
+
+    <Copy SourceFiles="..\PosnLibrary\bin\$(Platform)\$(Configuration)\net5.0-windows10.0.19041.0\Microsoft.Windows.SDK.NET.dll"
+          DestinationFolder="$(OutDir)"
+          UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
+    
+    <Copy SourceFiles="..\PosnLibrary\bin\$(Platform)\$(Configuration)\net5.0-windows10.0.19041.0\PosnLibrary.dll"
+          DestinationFolder="$(OutDir)"
+          UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
+  </Target>  
+  
+</Project>
+
+```
+
+## References
+Here are some resources that demonstrate authoring C#/WinRT components and the changes discussed above.
+1. https://github.com/AdamBraden/MyRandom
+2. https://github.com/microsoft/CsWinRT/tree/master/Samples/Net5ProjectionSample

```

Comment:
This sample shows generating the projection assembly for a C++/WinRT component, don't think it has to do with authoring?